pages/_app.tsx

```diff
@@ -41,15 +42,17 @@ function MyApp({ Component, pageProps }: AppProps) {
     <WagmiConfig client={wagmiClient}>
       <RainbowKitProvider modalSize="compact" chains={chains}>
         <ThemeProvider>
-          <SessionProvider>
-            <AdminProvider>
-              <ChatProvider>
-                <Component {...pageProps} />
-                <ToastContainer />
-                <Analytics />
-              </ChatProvider>
-            </AdminProvider>
-          </SessionProvider>
+          <MintProvider>
```

Comment:
I think `MintProvider` should only wrap around `{children}` of the `/mint` route right?

Can we undo changes to `app.tsx` and put `<MintProvider />` closer to where it's being used?